Apps/W1/HybridSL/test/src/SLBCGenProdPostingGroup.XmlPost.al

```diff
@@ -0,0 +1,75 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+namespace Microsoft.DataMigration.SL;
+
+using Microsoft.Finance.GeneralLedger.Setup;
+
+xmlport 47223 "SL BC Gen. Prod. Posting Group"
```

Comment:
These are again not valid test IDs. You should stay within the range 130.000 - 149.999. Take a look at the other objects you added previously. I re-IDed those too 🙂 The specified range for this test app is "from": 147600, "to": 147620. Try to stay within that range.

Reply:
After sync, the IDs were updated. All good now!